moses.lua

```diff
@@ -27,7 +27,7 @@ local function f_max(a,b) return a>b end
 local function f_min(a,b) return a<b end
 local function clamp(var,a,b) return (var<a) and a or (var>b and b or var) end
 local function isTrue(_,value) return value and true end
-local function iNot(value) return not value end
+local function iNot(_,value) return not value end
```

Comment:
we don't need to change this line

Reply:
because iNot() is using in _.reject() callback.

Reply:
`_.reject` is using `_.map` and `_.map` is using `function(v, k)`

Reply:
Right, so originally _map() is using (k, v), now it's (v,k).
That's why I change it to iNot(_,value).